lib/pagy/frontend.rb

```diff
@@ -10,10 +10,23 @@ class Pagy
   # see https://ddnexus.github.io/pagy/api/frontend#i18n
   I18n = eval(Pagy.root.join('locales', 'utils', 'i18n.rb').read) #rubocop:disable Security/Eval
 
+  module Helpers
+    # This works with all Rack-based frameworks (Sinatra, Padrino, Rails, ...)
+    def pagy_url_for(page, pagy, path_or_url=:path)
```

Comment:
```suggestion
    def pagy_url_for(page, pagy, path_or_url: :path)
```

Reply:
nicer, but Pagy is ruby 1.9+ compatible since version 2 (I needed to use it in legacy apps with legacy envs)